setup.py

```diff
@@ -13,18 +13,29 @@
     name='dawgz',
     version=dawgz.__version__,
     description='Directed Acyclic Workflow Graph Scheduling',
+    keywords='workflow-engine hpc slurm hpc-tools reproducible-science acyclic workflow graph scheduling',
     long_description=readme,
     long_description_content_type='text/markdown',
-    keywords='acyclic workflow graph scheduling',
-    author='François Rozet',
+    author='François Rozet, Joeri Hermans, Arnaud Delaunoy',
     author_email='francois.rozet@outlook.com',
     url='https://github.com/francois-rozet/dawgz',
     install_requires=required,
+    license='MIT license',
     packages=setuptools.find_packages(),
+    project_urls={
+        'Documentation': 'https://github.com/francois-rozet/dawgz',
+        'Source': 'https://github.com/francois-rozet/dawgz',
+        'Tracker': 'https://github.com/francois-rozet/dawgz/issues',
+    },
     classifiers=[
-        "Programming Language :: Python :: 3",
-        "License :: OSI Approved :: MIT License",
-        "Operating System :: OS Independent",
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: MIT License',
+        'Natural Language :: English',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
```

Comment:
Thinking of it, it is already mentioned below that `python_requires>=3.8`. It might not be necessary to enumerate them.

Reply:
ohhh these are classifiers so they help indexing the package, should keep these in